Wikipedia/Code/TalkPageDataController.swift

```diff
@@ -125,9 +137,11 @@ class TalkPageDataController {
                 
                 switch result {
                 case .success(let items):
+                    cache.talkPages = items
```

Comment:
Can you name this property `talkPageItems` instead of `talkPages`? So it will be:

`cache.talkPageItems = items`. I think it will be a little more clear.